nemo/collections/nlp/modules/common/huggingface/huggingface_decoder.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
typo: 2020 -> 2021

Reply:
Yes, no one wants to be stuck in 2020 :)

Reply:
Done.